vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -97,7 +91,7 @@ Use `ent help` to review the list of available commands
 > ⚠ RECOMMENDED FIRST STEP ⚠ :
   - Check the dependencies (ent check-env --help)
 ```
-Check a command's help text (`--help`) to view specific options and subcommands, e.g. `ent check-env --help`.
+Check a command's help text (`--help`) to view specific options and subcommands, e.g. `ent check-env --help`. For general configuration options related to ent itself, see `ent --help`.
```

Comment:
Check a command's help "text" >> details

Reply:
i think we should keep "help text" - that's the usual phrase to refer to help documentation